desktop/libs/notebook/src/notebook/conf.py

```diff
@@ -77,6 +77,15 @@ def _connector_to_interpreter(connector):
   }
 
 
+def displayName(analytics, name):
```

Comment:
We are not using analytics params anywhere.

Reply:
corrected and some changes in code.